consul/client.go

```diff
@@ -328,50 +321,34 @@ func (c *Client) localEvent(event serf.UserEvent) {
 
 // RPC is used to forward an RPC call to a consul server, or fail if no servers
 func (c *Client) RPC(method string, args interface{}, reply interface{}) error {
-	// Check the last rpc time
-	var server *serverParts
-	if time.Now().Sub(c.lastRPCTime) < clientRPCCache {
-		server = c.lastServer
-		if server != nil {
-			goto TRY_RPC
-		}
-	}
-
-	// Bail if we can't find any servers
-	c.consulLock.RLock()
-	if len(c.consuls) == 0 {
-		c.consulLock.RUnlock()
+	server := c.serverMgr.FindServer()
+	if server == nil {
+		c.logger.Printf("[ERR] consul: No healthy servers found in the server config")
```

Comment:
This feels like it will be spammy and "server config" makes me think I've done something wrong configuring things. I'd probably just drop this log message since this returns a pretty clear error to the caller.